pkg/apis/build/v1alpha2/build_types.go

```diff
@@ -63,10 +63,11 @@ type BuildSpec struct {
 	// +listType
 	Bindings Bindings `json:"bindings,omitempty"`
 	// +listType
-	Env       []corev1.EnvVar             `json:"env,omitempty"`
-	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
-	LastBuild *LastBuild                  `json:"lastBuild,omitempty"`
-	Notary    *NotaryConfig               `json:"notary,omitempty"`
+	Env                   []corev1.EnvVar             `json:"env,omitempty"`
+	ProjectDescriptorPath string                      `json:"projectDescriptorPath,omitempty"`
```

Comment:
Should this be `descriptorPath` to match the pack flag of `--descriptor`?

Reply:
going with `projectDescriptorPath` to disambiguate from a kp import descriptor